theurgy.lic

```diff
@@ -729,12 +725,24 @@ class TheurgyActions
     else
       DRCI.get_item_safe('incense', @theurgy_supply_container)
     end
+    
+    unless exists?('flint')
+      bput('drop my incense','You')
```

Comment:
Can we lower to feet instead of dropping? Or is there something else safer than dropping?

Reply:
We can but not really any major reason to?  Incense only costs a couple copper and you use up/buy it constantly.  One fail due to missing incense (I've never had it disappear in that 1 sec its on the ground) would be fixed on the very next running because you check supplies again.  Lowering to ground won't break anything but it could potentially cause a slight delay in leaving a room due to things atfeet.

Reply:
I guess it doesn't matter much for incense, I was just thinking that in general we try to do that for safety/to make sure we don't lose items. Are there any other 'incense' nouns you could end up with in your hand that are actually valuable?

Reply:
True. I can't think of anything off hand but better to future proof!